### ISSUE TITLE
Cannot find module './loader'

### ISSUE THREAD

**Issue by nvthuong1996** [nuxt/components#35]
**Describe the bug**

**To Reproduce**
Goto: https://codesandbox.io/s/nuxt-components-cou9k
Click button: Fork
Wating build
Error:
```
  Error: Cannot find module './loader'
  Require stack:
  - node_modules/@nuxt/components/dist/index.js
```

**Comment by pi0** [comment]
Hi. Would you please try removing yarn.lock/node_modules and installing dependencies again? Should be resolved by  v0.3.3

**Comment by nvthuong1996** [comment]
Are you sure you have tested before release
i can try fork from codesandbox. edit version to 0.3.3, delete yarn.lock. restart project. but it don't working. component not register exacly
I tried both on local with the same way and deleted node_modules

**Comment by pi0** [comment]
Hi @nvthuong1996. Indeed. There was a regression for dist introduced in 0.3.2 and fixed in 0.3.3 (regarding Cannot find module './loader')

You have to set `components: true` inside `nuxt.config` as well ([docs](https://github.com/nuxt/components#usage))

https://codesandbox.io/s/nuxt-components-3ons5?file=/nuxt.config.js
